### ISSUE TITLE
Double app bar transition

### ISSUE THREAD

**Issue by GreyAsteroid** [inorichi/tachiyomi#805]
The  app bar transition is made twice when selecting the first manga in your library after opening the app.
I cannot reproduce this with only 1 manga in my library, I've tried slowing down the animation speed as well as underclocking my device to slow it down. 
It only happens when using my full library which is 400+. Even when I have it filtered to unread leaving 60~.
This is while using gridview with 4 columns. 

Version: r1164
OS: Lineage 14.1 Android 7.1.2

**Comment by inorichi** [comment]
Do you use multiple categories?

**Comment by GreyAsteroid** [comment]
@inorichi No, only the default category so the app bar is not expanded by default

**Comment by inorichi** [comment]
I tried with a library of 21 entries in the default category. The expand method (with the animation) is called exactly once when clicking any of the entries, regardless if the app was just launched.

**Comment by GreyAsteroid** [comment]
I'll start from a clean install and see if I can find out how to reproduce it.

**Comment by GreyAsteroid** [comment]
Alright, the long press shortcut to add manga to your library made this a lot faster than I thought it was going to be. 

I added the top 68 manga from kissmanga to my library and I was not able to reproduce this issue.
I then added a bit more totaling 120 manga in my library and then I was able to reproduce this issue. 
I then deleted 74 manga from my library leaving me with 46 and I was still able to reproduce this issue?

It affects both grid view of any size as well as list view. 
It only happens on the first manga you select.
Occasionally you'll get the normal single transition, happened once out of the 5 times I tried.

Another thing to note (I should probably file a separate bug report for this), the white splash screen? that shows up as soon as you open the app doesn't last as long the more manga you have in your library. Meaning that you're sent to an empty Library with an expanded app bar until the app bar transition happens and the list of manga loads.
120 was enough to trigger this to a degree.
On my regular install with 400+ manga, the "splash screen" no longer shows up at all.
